_legacy_server/api/auth/utils/resetPassword.ts

```diff
@@ -1,5 +1,5 @@
 import { User } from '@core/auth/user'
-import { createI18nPromise } from '@common/i18n/i18nFactory'
+import { createI18nPromise } from '@common/../../../../i18n/i18nFactory'
```

Comment:
what's this import?

Reply:
hmm, automatically updated by intellij I think, fixed